tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/common/SmokeTestBase.java

```diff
@@ -158,24 +156,6 @@ protected static final void recreateBrokerDirectory(final String homeInstance) {
       recreateDirectory(homeInstance + "/log");
    }
 
-
-   public boolean waitForServerToStart(String uri, String username, String password, long timeout) throws InterruptedException {
-      long realTimeout = System.currentTimeMillis() + timeout;
-      while (System.currentTimeMillis() < realTimeout) {
-         try (ActiveMQConnectionFactory cf = ActiveMQJMSClient.createConnectionFactory(uri, null)) {
-            cf.createConnection(username, password).close();
-            System.out.println("server " + uri + " started");
-         } catch (Exception e) {
-            System.out.println("awaiting server " + uri + " start at ");
-            Thread.sleep(500);
-            continue;
-         }
-         return true;
-      }
-
-      return false;
-   }
-
```

Comment:
Is this change actually related to the ZooKeep & Curator upgrade? It looks like an unrelated removal of an unused method.

Reply:
looks like actual logic is moved to ServerUtil and this method confuses when looking for corresponding check

Reply:
Thats fine, but Justin's correct point is it shouldn't be in the same commit as it isnt related.

Reply:
moved to additional PR